var/spack/repos/builtin/packages/rocprim/package.py

```diff
@@ -104,6 +105,7 @@ class Rocprim(CMakePackage):
         "5.0.2",
         "5.1.0",
         "5.1.3",
+        "5.2.0",
```

Comment:
self.define("CMAKE_MODULE_PATH", self.spec["hip"].prefix.cmake),  needs to change for rocprim for 5.2.0 as cmake is installed at a different path . check for others also.

Reply:
Added CMAKE_MODULE_PATH for ginkgo with the right path.
Restricting it to 5.1.3 in hipblas, hipfft, hipsparse and rocprim since it doesn't have any impact.